wireapp/wire-android
# fix: memory leak video renderer

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

There is a memory leak you are in a video call on try to kill the app.
The issue is visible on LeakCanary

### Causes (Optional)

VideoRenderer not cleared

### Solutions

Destroy renderer when onDestroy() lifecycle happens

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

On a video call try to kill the app (by swiping it to top),  you should not see the memory leak on LeakCanary

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
